src/routes/about/+page.js

```diff
@@ -0,0 +1,352 @@
+        ]
+      }
+    ]
+  },
+  {
+    id: 17051931,
+    name: "C.A.M. Gerlach",
+    role: "Core developer",
+    avatar_url: "/assets/authors/camgerlach/pic.webp",
+    tooltip: [
+      {
+        list: [
+          "Docs maintainer",
+          "Technical writer"
+        ]
+      }
+    ]
+  },
+  {
+    id: 6740194,
+    name: "Quentin Peter",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Re-architecture communications between Spyder and Spyder-kernels",
+          "Creator of the Debugger pane",
+          "Many improvements to the IPython console"
+        ]
+      }
+    ]
+  },
+  {
+    id: 9618975,
+    name: "Ryan Clary",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Maintainer of the standalone installers"
+        ]
+      }
+    ]
+  },
+  {
+    id: 7941918,
+    name: "Jitse Niesen",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "External plugins maintainer"
+        ]
+      }
+    ]
+  },
+  {
+    id: 42411448,
+    name: "Juan Sebastian Bautista",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Improvements to the Windows installer, QtConsole and Spyder's UI/UX"
+        ]
+      }
+    ]
+  },
+  {
+    id: 5204788,
+    name: "Hendrik D. Louzada",
+    role: "Junior Developer",
+    tooltip: [
+      {
+        list: [
+          "Backend for the Remote client plugin"
+        ]
+      }
+    ]
+  },
+  {
+    id: 5027583,
+    name: "Andrés Montoya",
+    role: "Grpahic and web designer",
+    tooltip: [
+      {
+        list: [
+          "Developer and maintainer of this website",
+          "UI/UX improvements",
+          "Social media graphics designer"
+        ]
+      }
+    ]
+  },
+];
+
+const pastContributors = [
+  {
+    id: 1311787,
+    name: "Pierre Raybaut",
+    role: "Spyder Creator",
+    tooltip: [
+      {
+        list: [
+          "Started Spyder in 2009",
+          "Lead maintainer until 2013",
+        ]
+      }
+    ]
+  },
+  {
+    id: 1878982,
+    name: "Edgar Margffoy",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "LSP support for the entire application",
+          "Re-architecture plugin registration and interactions",
+          "New architecture for the Run plugin",
+          "Add global registries for actions, menus and toolbars",
+        ]
+      }
+    ]
+  },
+  {
+    id: 50221806,
+    name: "Isabela Presedo-Floyd",
+    role: "Designer",
+    tooltip: [
+      {
+        list: [
+          "New color system and palette for the application",
+          "Logo redesign",
+          "Many UI/UX improvements"
+        ]
+      }
+    ]
+  },
+  {
+    id: 3627835,
+    name: "Gonzalo Peña-Castellanos",
+    role: "Core developer",
+    tooltip: [
+      {
+        list: [
+          "Redesign of the plugins API",
+          "Creator of Qtpy",
+          "Test and improve configuration system",
```

Comment:
```suggestion
          "Improved configuration system",
```

Its a little odd to just say someone tested something as one of their biggest contributions. Or do you mean

```suggestion
          "Improved configuration system and added tests",
```